standard/standard-library.md

```diff
@@ -658,3 +652,400 @@ namespace System.Runtime.CompilerServices
     }
 }
 ```
+
+## C.4 Format Specifications
+
+The meaning of the formats, as used in interpolated string expressions (§interpolated-string-expressions), are defined in ISO/IEC 23271:2012. For convenience the following text is copied from the description of `System.IFormatable`.
+
+**This text is informative.**
+
+A *format* is a string that describes the appearance of an object when
+it is converted to a string. Either standard or custom formats can be used. A
+standard format takes the form *Axx*, where *A* is a single
+alphabetic character called the *format specifier*
+, and *xx* is an integer between zero and 99 inclusive, called the *precision specifier*. The format specifier controls the type
+of formatting applied to the value being represented as a string. The
+*precision specifier* controls the number 
```

Comment:
(I appreciate this is a copy/paste job, and we may well not want to bother changing it at all.)

Reply:
I too pondered over how much to change this. The original style and formatting doesn't even fit the rest of the CLI Standard, but then it is distributed as an XML file that was mechanically produced from CLR documentation so it wasn't really noticable. I mainly did a mechanical translation into markdown and then a few hand tweaks so it's still recognisable as the source material.

This is the largest single block of format specs I spotted but it is about numeric types, the closing para directs the reader to `Enum` and `DateTime`. How much to include?

I think TG2 needs to ask whether it should foster (not adopt) the whole of the standard library docs while TG3 is in hibernation, to be returned when TG3 wakes up. That would of course address the "how much" issue, and would allow TG2 to remove §C.3 as it would be empty and TG2 would no longer have to fudge and pretend these the types in §C.3 have been Standardised…

Reply:
Hmm. I'd be concerned unless we can expand the membership in TG2, including some people who have more expertise in this area.